lib/agoneum/web/templates/layout/app.html.slim

```diff
@@ -0,0 +1,23 @@
+doctype html
```

Comment:
Technically either is fine. [Slime](https://github.com/slime-lang/phoenix_slime) was inspired by [Slim](http://slim-lang.com/) - we should pick one though and stick with that. I've left them with `.slim` since Spacemacs seems to pick those up just fine but if this is problematic for you - let me know and we can decide on which convention to use.